modin/core/execution/ray/implementations/cudf_on_ray/dataframe/dataframe.py

```diff
@@ -141,9 +142,17 @@ def mask(
 
         Notes
         -----
-        If both `row_labels` and `row_positions` are set, `row_labels` will be used.
-        The same rule applied to `col_labels` and `col_positions`.
+        If both row_labels and row_positions are provided, a ValueError is raised.
+        The same rule applies for col_labels and col_positions.
         """
+        if not (row_labels is None or row_positions is None):
+            raise ValueError(
+                "Both row_labels and row_positions were provided - please provide only one of row_labels and row_positions."
+            )
```

Comment:
This check is presented several times in different places. Can we unify this in a separate func maybe?